cli/proc_state.rs

```diff
@@ -676,6 +677,25 @@ impl SourceMapGetter for ProcState {
   }
 }
 
+pub fn import_map_from_text(
+  specifier: &Url,
+  json_text: &str,
+) -> Result<ImportMap, AnyError> {
+  let result = ImportMap::from_json_with_diagnostics(specifier, json_text)?;
+  if !result.diagnostics.is_empty() {
+    warn!(
+      "Import map diagnostics:\n{}",
+      result
+        .diagnostics
+        .into_iter()
+        .map(|d| format!("  - {}", d))
+        .collect::<Vec<_>>()
+        .join("\n")
+    )
+  }
```

Comment:
This previously was occurring within the import_map crate and it was doing `info!`. It seems more appropriate for this to be `warn!` though?